solidity/test/bridge/Bridge.test.ts

```diff
@@ -4849,6 +4849,833 @@ describe("Bridge", () => {
+                  data.redemptionRequests[0].amount
+                )
+                const currentRedeemerBalance = await bank.balanceOf(
+                  data.redemptionRequests[0].redeemer
+                )
+                expect(currentRedeemerBalance).to.be.equal(
+                  expectedRedeemerBalance
+                )
+              })
+
+              it("should remove the request from the pending redemptions", async () => {
+                const redemptionKey = buildRedemptionKey(
+                  data.wallet.pubKeyHash,
+                  data.redemptionRequests[0].redeemerOutputScript
+                )
+                const request = await bridge.pendingRedemptions(redemptionKey)
+
+                expect(request.redeemer).to.be.equal(
+                  "0x0000000000000000000000000000000000000000"
+                )
+                expect(request.requestedAmount).to.be.equal(0)
+                expect(request.treasuryFee).to.be.equal(0)
+                expect(request.txMaxFee).to.be.equal(0)
+                expect(request.requestedAt).to.be.equal(0)
+              })
+
+              it("should add the request to the timed-out redemptions", async () => {
+                const timedOutRequest = await bridge.timedOutRedemptions(
+                  buildRedemptionKey(
+                    data.wallet.pubKeyHash,
+                    data.redemptionRequests[0].redeemerOutputScript
+                  )
+                )
+
+                expect(timedOutRequest.redeemer).to.be.equal(
+                  data.redemptionRequests[0].redeemer
+                )
+                expect(timedOutRequest.requestedAmount).to.be.equal(
+                  redemptionRequest.requestedAmount
+                )
+                expect(timedOutRequest.treasuryFee).to.be.equal(
+                  redemptionRequest.treasuryFee
+                )
+                expect(timedOutRequest.txMaxFee).to.be.equal(
+                  redemptionRequest.txMaxFee
+                )
+                expect(timedOutRequest.requestedAt).to.be.equal(
+                  redemptionRequest.requestedAt
+                )
+              })
+
+              it("should change the wallet's state to MovingFunds", async () => {
+                const wallet = await bridge.getWallet(data.wallet.pubKeyHash)
+                expect(wallet.state).to.be.equal(walletState.MovingFunds)
+              })
+
+              it("should set the wallet's move funds requested timestamp", async () => {
+                const wallet = await bridge.getWallet(data.wallet.pubKeyHash)
+                expect(wallet.moveFundsRequestedAt).to.be.equal(
+                  await lastBlockTime()
+                )
+              })
+
+              it("should not emit WalletClosed events", async () => {
+                // Since the wallet has UTXO set it should be not be closed but
+                // marked as zMovingFunds` and we should not see any `WalletClosed`
```

Comment:
Typo `zMovingFunds`

Reply:
Done in https://github.com/keep-network/tbtc-v2/commit/a769909bce54058695ce018258de2ada00bb6175.